en-US/Samples/GpioTestTool.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: sample
```

Comment:
Please update the header to follow the new template
https://github.com/ms-iot/content/blob/develop/Resources/contribute/template/sample-template.md

Including a link to the gpiotesttool on GitHub.com/samples.  (i'll make sure that they go live at the exact same time)

The link would be www.github.com/ms-iot/samples/gpiotesttool